### ISSUE TITLE
Regression with property setter in typing stub (1.1.297) (matplotlib Colorbar)

### ISSUE THREAD

**Issue by jenshnielsen** [microsoft/pyright#4743]
**Describe the bug**

Type checking a piece of code that sets a property on a class with types defined in https://github.com/microsoft/python-type-stubs
results in an unexpected error ```Property "formatter" has no defined setter``` for a type stub that does have the setter defined.

In the specific example I am setter the formatter property on a matplotlib Colorbar. The type stub that I am checking against has the property and setter defined https://github.com/microsoft/python-type-stubs/blob/main/matplotlib/colorbar.pyi#L62

This does not happen with a class defined in the current module. See the additional example below.

**To Reproduce**

Type check the attached code with 1.1.297

**Expected behavior**

No errors

**Screenshots or Code**

```python
from matplotlib.colorbar import Colorbar
from matplotlib.ticker import Formatter


class DummyColorBar:


    def __init__(self) -> None:
        ...


    @property
    def formatter(self) -> Formatter | None:
        ...

    @formatter.setter
    def formatter(self, formatter: Formatter) -> None:
        ...

def set_formatter(cax: Colorbar, formatter: Formatter) -> None:
    cax.formatter = formatter # <- error is triggered here


def set_formatter_on_dummy(cax: DummyColorBar, formatter: Formatter) -> None:
    cax.formatter = formatter # but not here
```

```
❯ pyright .\debug_formatter.py
No configuration file found.
No pyproject.toml file found.
Assuming Python platform Windows
Searching for source files
Found 1 source file
pyright 1.1.297
C:\Users\jenielse\source\repos\debugpyright\debug_formatter.py
  C:\Users\jenielse\source\repos\debugpyright\debug_formatter.py:21:9 - error: Cannot assign member "formatter" for type "Colorbar"
    Property "formatter" has no defined setter (reportGeneralTypeIssues)
1 error, 0 warnings, 0 informations
Completed in 1.72sec
```

**VS Code extension or command-line**
Command-line 1.1.297 (works fine with 1.1.296)

**Additional context**

https://github.com/microsoft/python-type-stubs at current main https://github.com/microsoft/python-type-stubs/commit/4d3d9060158d8349ca2d8f5f6cea0ea58a7017d9 cloned into typings folder next to this file.

**Comment by erictraut** [comment]
Thanks for the bug report and apologies for the regression. This will be fixed in the next release.

**Comment by NeilGirdhar** [comment]
Not sure if this is related, but I'm having a property setter issue as well:
```python
import pandas as pd
data_frame = pd.DataFrame()
data_frame.columns = pd.Index(['x', 'y'])  # error: Cannot assign member "columns" for type "DataFrame"
```
The type stub is [here](https://github.com/pandas-dev/pandas-stubs/blob/6829a993697f8c5d65b721ca1cec3980ac79e69c/pandas-stubs/core/frame.pyi#L1362).

**Comment by erictraut** [comment]
This regression is bad enough that I've decided to do a quick-spin release to fix it. Version 1.1.298 will be out shortly.

**Comment by erictraut** [comment]
Version 1.1.298 is now published. Please upgrade, and this problem should be fixed.

**Comment by jenshnielsen** [comment]
Thanks for the super fast response @erictraut
